index.js

```diff
@@ -121,7 +121,7 @@ module.exports = function i18nPlugin({ types: t }) {
                                     previousNode,
                                     currentNode,
                                 );
-                            });
+                            }, '');
```

Comment:
There was no initial value set for the reduce function.

In the case of there only being a token then the array the reducer was being called on would only contain one item:

```
['{name}']
```

When there is no initial value and only one item in the array reduce does not execute.

![Screenshot 2023-08-18 at 13 35 27](https://github.com/Skyscanner/babel-plugin-transform-i18n/assets/25118364/73bc1ce9-48fa-47d0-a76f-9fd1f36c31f8)

This is a problem, as `getNodeForValue` called on every item that reduce executes on is also the function to transform it (from `{name}` to `Rob`).

By adding an initial value the reducer uses the first item in the array as its first current value, setting its first run to index 0, and allowing us inside the reducer.

![Screenshot 2023-08-18 at 13 41 46](https://github.com/Skyscanner/babel-plugin-transform-i18n/assets/25118364/a7879abb-9ff4-48e4-a1d8-2e0b5118e709)

We then need to make sure to do the 'previousNode' check on the initial value (moving its index check from 1 to 0, as when using an initial value it starts at 0, when not using an initial value it starts at 1).

> If initialValue is specified, callbackFn starts executing with the first value in the array as currentValue. If initialValue is not specified, accumulator is initialized to the first value in the array, and callbackFn starts executing with the second value in the array as currentValue

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce